### PR TITLE
Plans page redirect, display fixes

### PR DESCRIPTION
**Pull request by LMNTL**
## Checklist

1. [ ] If you've added code that should be tested, add tests
2. [ ] If you've changed APIs, update (or create!) the documentation
3. [X] Ensure the tests pass
4. [X] Make sure that your code lints and that you've followed [our coding style](https://github.com/kobotoolbox/kpi/blob/master/CONTRIBUTING.md)
5. [X] Write a description of your work suitable for publishing on [our forum](https://community.kobotoolbox.org/tag/release-notes)
6. [ ] Mention any related issues in this repository (as #ISSUE) and in other repositories (as kobotoolbox/other#ISSUE)
7. [ ] Open an issue in the [docs](https://github.com/kobotoolbox/docs/issues/new) if there are UI/UX changes

## Description

Redirect users away from `/#/account/plan` if Stripe isn't enabled or there are no products available. Fix some display issues on the Plans page.

## Notes
You should be redirected to `/#/account/settings/` without Stripe or products.

Display changes on the plans page:
* The interval toggle (Annual/Monthly) now defaults to Monthly if the user has the Community plan. If the user has a paid subscription, it should still initially show Annual if the user has an annual subscription.
* The 'your plan' indicator now shows in both the Annual and Monthly views. Previously it only showed in the Monthly view.

### CODE REVIEW

**Comment by srartese** [comment]
A few notes that don't have anywhere specifically to go: 
- Check against the linter (I did leave one from previous work there about the any which is on my list to go back so feel free to ignore that one) 
- When testing there was a bug that I'm don't think is possible when live but I wanted to point out incase. I had multiple subscriptions on an account because of not updated stripe immediately which meant the banner was showing on both monthly and yearly.

File: jsapp/js/account/plan.component.tsx [hunk 1]
```diff
@@ -29,6 +29,8 @@ import LoadingSpinner from 'js/components/common/loadingSpinner';
 import {notify} from 'js/utils';
 import {BaseProduct} from 'js/account/subscriptionStore';
```
Comment (srartese) [inline jsapp/js/account/plan.component.tsx:30]: This isn't used anywhere

File: jsapp/js/account/plan.component.tsx [hunk 2]
```diff
@@ -29,6 +29,8 @@ import LoadingSpinner from 'js/components/common/loadingSpinner';
...
 import envStore, {FreeTierThresholds, FreeTierDisplay} from 'js/envStore';
```
Comment (srartese) [inline jsapp/js/account/plan.component.tsx:31]: FreeTierDisplay also looks to not be used anywhere

**Review by srartese — CHANGES_REQUESTED** [review]

**Comment by LMNTL** [comment]
>     * When testing there was a bug that I'm don't think is possible when live but I wanted to point out incase. I had multiple subscriptions on an account because of not updated stripe immediately which meant the banner was showing on both monthly and yearly.

Did you have a monthly and yearly subscription for the same plan? If so, that would be expected behavior - you have both types of subscription, and the frontend should reflect that. But if not, I have some bug-hunting to do 🙂

**Review by srartese — APPROVED** [review]
Yea I had two subscribed and I canceled one but it is still active technically. Sounds like its not a bug 🎉

**Pull request merged**